pkg/handler/common/provider/azure.go

```diff
@@ -299,6 +299,14 @@ func isValidVM(sku compute.ResourceSku, location string) bool {
 }
 
 func AzureSize(ctx context.Context, quota kubermaticv1.MachineDeploymentVMResourceQuota, subscriptionID, clientID, clientSecret, tenantID, location string) (apiv1.AzureSizeList, error) {
+	// https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu
+	gpuInstanceFamilies := map[string]int32{"Standard_NC6": 1, "Standard_NC12": 2, "Standard_NC24": 4, "Standard_NC24r": 4,
```

Comment:
i would move this a package var so the map is not initialized every time AzureSize is called.

Reply:
done